### DISCUSSION TITLE
dynamic home position

### DISCUSSION THREAD

**Discussion by anti-vaxxer** [iNavFlight/OpenTX-Telemetry-Widget#159] [General]
Hi everyone,
I'm using the INAV Lua Telemetry Widget on my TX16S running EdgeTX. I heard it's possible to add GPS to TX16S, so I want to know if I add GPS will the 'home position' shown on the widget's map update in relation to my model's position when I move with my radio? or is the home position permanently fixed at where I armed the model?
This would be useful for finding lost model or if flying from moving car/boat etc.
I asked about this first in [RCGroups](https://www.rcgroups.com/forums/showthread.php?3916381-Official-EdgeTX-Discussion-Thread&postid=52950631) but got a bunch of unhelpful comments so I'm asking here.

**Comment by stronnag** [comment]
The home position is that reported by telemetry.

**Reply by anti-vaxxer** [reply]
Okay so home position is fixed at where the model is armed for the first time right?

Then I still want to know whether it's possible to make a feature for the Widget so the handset's internal GPS can be used for a dynamic home position?

**Comment by stronnag** [comment]
If someone generates a PR it will be considered, but taking into account the title of this widget "INAV **Telemetry** Widget", and the run time constraints of the platform, then, at least this maintainer would need evidence of comprehensive regression testing  on this and other devices before such a PR could be merged.

If you want a fully capable ground station, you might consider using one, rather than trying to coerce this widget to be something that it is not.

**Comment by anti-vaxxer** [comment]
Just wanted to share, years ago someone did an OpenTX Lua Telemetry Script with dynamic Pilot/Home position feature with GPS installed in the radio.
https://rcdiy.ca/gps-directions-home/
https://forum.flitetest.com/index.php?threads/gps-distance-trip-find-return-home-lua-script.34085/
